sound/soc/sof/core.c

```diff
@@ -306,6 +306,50 @@ static int sof_machine_check(struct snd_sof_dev *sdev)
 #endif
 }
 
+static int sof_dsp_start(struct snd_sof_dev *sdev)
```

Comment:
@lyakh but the gues would init the IPC? that doesnt seem correct right?

Reply:
@ranj063 it's correct - the guest also needs the IPC infrastructure, it uses it to communicate with the host via VirtIO queues